tdp/cli/utils.py

```diff
@@ -128,6 +128,6 @@ def vars(func: FC) -> FC:
         "--vars",
         envvar="TDP_VARS",
         required=True,
-        type=click.Path(resolve_path=True, path_type=Path),
+        type=click.Path(resolve_path=True, path_type=Path, exists=True),
```

Comment:
I think that the initial situation when we use tdp-lib is that the directory tdp_vars doesn't exist.
But, when executing ```tdp init``` with no directory for tdp_vars, I have an error : 
```
Error: Invalid value for '--vars': Path './tdp_vars' does not exist.
```

Reply:
Thanks! I fixed it!